### PR TITLE
75 carousel functionality

### PR DESCRIPTION
**Pull request by allanviguilla**
feature: carousel, go to previous image/go to next image

[trello ticket](https://trello.com/c/iG0RRlc8/75-product-overview-carousel-functionality-l)

### CODE REVIEW

**Review by hepnerthomas — CHANGES_REQUESTED** [review]
@allanviguilla Please merge in the changes from the `master` branch so we don't lose prior!

File: client/src/components/ProductOverview/Carousel/Carousel.jsx [hunk 1]
```diff
@@ -1,30 +1,46 @@
...
-    backgroundPosition: 'center',
-    backgroundSize: '100%',
-    backgroundRepeat: 'no-repeat',
-    backgroundImage: 'url(https://via.placeholder.com/800x1000)',
+  const [productImages, setProductImages] = useState([]);
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [currentImage, setCurrentImage] = useState('');
+
+  useEffect(() => {
+    axiosConfig.get('/products/65633/styles')
```
Comment (hepnerthomas) [inline client/src/components/ProductOverview/Carousel/Carousel.jsx]: pull `productId` from App state

**Review by hepnerthomas — CHANGES_REQUESTED** [review]
Behavior is as desired, but `productId` needs to be pulled from global state.

Will approve and merge once this change is made.

**Comment by allanviguilla** [comment]
@hepnerthomas done! Changed productId in App state and passed props down to my component. Ready for review and merge

**Review by hepnerthomas — APPROVED** [review]
Requested changes were made - approved!

**Pull request merged**